string key
---
bool success
string metadata
string value
